### PR TITLE
Add code files for Databricks lambda blog

### PR DESCRIPTION
**Pull request by mortenm-bov**

### CODE REVIEW

File: bouvet-deler/databricks_lambda/databricks_lambda_case1.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+# MAGIC # Using Lambda in Databricks - Case 1
+# MAGIC 
+# MAGIC This notebooks gives a simple example of using lambda in Databricks. For more information see this article (add link).
+# MAGIC 
+# MAGIC Author: Morten Mjelde <morten.mjelde@bouvet.no>
+
+# COMMAND ----------
+
+# Define DataSet1
+from pyspark.sql import Row
```
Comment (merca) [inline bouvet-deler/databricks_lambda/databricks_lambda_case1.py:12]: can you put some air in here

File: bouvet-deler/databricks_lambda/databricks_lambda_case1.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+)
+
+# COMMAND ----------
+
+# Define transformations
+from pyspark.sql.functions import concat, col, DataFrame, lit
+
+def addInitials(df: DataFrame) -> DataFrame:
+    """Add initials column."""
+    df_new = df.withColumn("intials", concat(col("firstName").substr(0,1), col("lastName").substr(0,1)))
```
Comment (merca) [inline bouvet-deler/databricks_lambda/databricks_lambda_case1.py]: out of personal curiosity: would `col("lastName")[:1]` work here?

File: bouvet-deler/databricks_lambda/databricks_lambda_case2.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+# MAGIC Author: Morten Mjelde <morten.mjelde@bouvet.no>
+
+# COMMAND ----------
+
+# Define DataSet3
+from pyspark.sql import Row
+
+DataSet3 = spark.createDataFrame(
+    [
+        Row(location="Oslo", people=[Row(firstName="Harald", lastName="Haraldsen"), Row(firstName="Mark", lastName="Hunter")]),
```
Comment (merca) [inline bouvet-deler/databricks_lambda/databricks_lambda_case2.py]:
```suggestion
        Row(location="Oslo", people=[
                Row(firstName="Harald", lastName="Haraldsen"), 
                Row(firstName="Mark", lastName="Hunter")
            ]),
```
Maybe for readabilty?

File: bouvet-deler/databricks_lambda/databricks_lambda_case2.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+)
+
+# COMMAND ----------
+
+# Define DataSet4
+from pyspark.sql import Row
+
+DataSet4 = spark.createDataFrame(
+    [
+        Row(location="Copenhagen", people=[Row(firstName="Roger", nobleName="of house Henriksen"), Row(firstName="Hans", nobleName="of house Dale")]),
```
Comment (merca) [inline bouvet-deler/databricks_lambda/databricks_lambda_case2.py]: Same for this. Did you run black on the code?

File: bouvet-deler/databricks_lambda/databricks_lambda_case2.py [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+)
+
+# COMMAND ----------
+
+# Define transformations
+from pyspark.sql.functions import concat, col, DataFrame, lit, explode
+
+def explodePeople(df: DataFrame) -> DataFrame:
+    """Explode people column."""
+    return df.withColumn("ex_people", explode("people")).select("location", "ex_people.*")
```
Comment (merca) [inline bouvet-deler/databricks_lambda/databricks_lambda_case2.py]:
```suggestion
    return df.withColumn("expl_people", explode("people")).select("location", "expl_people.*")
```
ex_people gisves same feeling as terminated_people 😃

File: bouvet-deler/databricks_lambda/databricks_lambda_case2.py [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+    df_new = df.withColumn("intials", concat(col("firstName").substr(0,1), col("lastName").substr(0,1)))
+    return df_new
+
+def addfullName(df: DataFrame) -> DataFrame:
+    """Add full name column."""
+    return df.withColumn("fullName", concat(col("firstName"), lit(" "), col("lastName")))
+
+def nameTransformation(data: DataFrame, correctNameColumns: callable = lambda df: df) -> DataFrame:
+    """Perform all transformations."""
+    data = explodePeople(data)
```
Comment (merca) [inline bouvet-deler/databricks_lambda/databricks_lambda_case2.py:76]: have to love the naming here 🤣 and the column over should then be `withColumn("exploded_people",......)`

**Review by merca — COMMENTED** [review]
Looks very good!

File: bouvet-deler/databricks_lambda/databricks_lambda_case1.py (hunk 1 above)
Comment (mortenm-bov) [reply bouvet-deler/databricks_lambda/databricks_lambda_case1.py:12]: I think it looks better as is. Adding a black line makes it harder to read imo.

File: bouvet-deler/databricks_lambda/databricks_lambda_case2.py (hunk 3 above)
Comment (mortenm-bov) [reply bouvet-deler/databricks_lambda/databricks_lambda_case2.py]: Fixed.

File: bouvet-deler/databricks_lambda/databricks_lambda_case2.py (hunk 4 above)
Comment (mortenm-bov) [reply bouvet-deler/databricks_lambda/databricks_lambda_case2.py]: Fixed.

File: bouvet-deler/databricks_lambda/databricks_lambda_case2.py (hunk 5 above)
Comment (mortenm-bov) [reply bouvet-deler/databricks_lambda/databricks_lambda_case2.py]: Good point. Fixed.

**Review by merca — APPROVED** [review]

**Pull request merged**